renderdoc/driver/metal/metal_render_command_encoder.cpp

```diff
@@ -76,6 +77,94 @@ void WrappedMTLRenderCommandEncoder::setRenderPipelineState(WrappedMTLRenderPipe
   }
 }
 
+template <typename SerialiserType>
+bool WrappedMTLRenderCommandEncoder::Serialise_setVertexBuffer(SerialiserType &ser,
+                                                               WrappedMTLBuffer *buffer,
+                                                               NS::UInteger offset,
+                                                               NS::UInteger index)
+{
+  SERIALISE_ELEMENT_LOCAL(RenderCommandEncoder, this);
+  SERIALISE_ELEMENT(buffer).Important();
+  SERIALISE_ELEMENT(offset).Important();
```

Comment:
Going by the convention for other APIs, the buffer offset like this is not considered 'important' for this type of thing. It's still available upon expanding and digging in, but the offset is more often noise compared to seeing what slot is being bound and which buffer.